src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/ElytraCap.java

```diff
@@ -41,7 +41,7 @@ public boolean isDamageable() {
     }
 
     @Override
-    public void damageItem(Player p, ItemStack item) {
+    public void damageItem(@Nonnull Player p, ItemStack item) {
```

Comment:
nullability for `item` too

Reply:
`item` is `Nullable` according to `DamageableItem.java` at line `48`

Reply:
So, provide @Nullable